pkg/substitution/substitution_test.go

```diff
@@ -80,39 +80,13 @@ func TestSetCloudControllerImage(t *testing.T) {
 
 	for _, tc := range tc {
 		t.Run(tc.name, func(t *testing.T) {
-			deploy := &v1.Deployment{
-				Spec: v1.DeploymentSpec{
-					Template: corev1.PodTemplateSpec{
-						Spec: corev1.PodSpec{
-							Containers: tc.containers,
-						},
-					},
-				},
-			}
-
-			ds := &v1.DaemonSet{
-				Spec: v1.DaemonSetSpec{
-					Template: corev1.PodTemplateSpec{
-						Spec: corev1.PodSpec{
-							Containers: tc.containers,
-						},
-					},
-				},
-			}
-
 			pod := &corev1.Pod{
```

Comment:
Why are we still using a pod? Should we not just initialise the pod spec? The pod wrapper here doesn't add anything

Reply:
okay, why not...